src/common/datasource/src/test_util.rs

```diff
@@ -77,6 +78,32 @@ pub fn test_basic_schema() -> SchemaRef {
     Arc::new(schema)
 }
 
+pub fn test_orc_schema() -> SchemaRef {
```

Comment:
For large schema, It's better to infer the schema from the file?

https://github.com/GreptimeTeam/greptimedb/blob/b9bee9128de2c580bf2ec1637813eef9b51ad490/src/common/datasource/src/file_format/orc.rs#L114-L126